internal/dump/tar_test.go

```diff
@@ -151,7 +164,17 @@ func checkTar(t *testing.T, testDir string, srcTar *bytes.Buffer) error {
 			if filepath.Base(hdr.Name) != filebase {
 				return fmt.Errorf("foldernames don't match got %v want %v", filepath.Base(hdr.Name), filebase)
 			}
-
+			if !strings.HasSuffix(hdr.Name, "/") {
+				return fmt.Errorf("foldernames must end with separator got %v", hdr.Name)
+			}
+		} else if hdr.Typeflag == tar.TypeSymlink {
```

Comment:
Nitpick: is this the right moment to turn this into a switch statement?

Reply:
Yep, good point!